cluster/saltbase/salt/cluster-autoscaler/cluster-autoscaler.manifest

```diff
@@ -27,9 +27,18 @@
                 "name": "cluster-autoscaler",
                 "image": "gcr.io/google_containers/cluster-autoscaler:v0.5.0-beta1",
                 "command": [
-                    "/bin/sh",
-                    "-c",
-                    "./cluster-autoscaler --kubernetes=http://127.0.0.1:8080?inClusterConfig=f --v=4 {{params}} 1>>/var/log/cluster-autoscaler.log 2>&1"
+                    "./run.sh",
+                    "--kubernetes=http://127.0.0.1:8080?inClusterConfig=f",
+                    "--v=4",
+                    "--stderrthreshold=info",
+                    "--write-status-configmap=true",
+                    "{{params}}"
+                ],
+                "env": [
+                    {
+                        "name": "LOG_OUTPUT",
+                        "value": "/var/log/cluster-autoscaler.log"
```

Comment:
This file is mounted from the machine disk. I guess that there is some log rotation policy for /var/logs on the master (but can doublecheck).